### PR TITLE
gallery: Add gallery subdomain

### PR DESCRIPTION
**Pull request by juliaogris**
Add gallery subdomain to evy[stage|test].dev domains. It all started with me
wanted to refer to the lab from the docs now that we have got at least parity
with what we had on the GitHub wiki. I then realized I don't want anymore links
to the GitHub wiki so I removed them all in favor of links to lab.evy.dev and
gallery.evy.dev. The gallery at this stage is only a redirect to the GitHub
wiki but I plan to set up a proper gallery. At some point.

I've also added a link to the new cheat-sheet (which still needs a second page).

### CODE REVIEW

File: e2e/docs/test.js-snapshots/comment-page-chromium-docker.png
Comment (juliaogris) [inline e2e/docs/test.js-snapshots/comment-page-chromium-docker.png:1]: I'm not sure what's different here, but the tests fail without the update.
The entry paragraph to this page has changed, so maybe related to that.

File: docs/syntax-by-example.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
-The following examples will help you understand the syntax of Evy. For a
-more formal definition of the syntax, see the
-[Language Specification](spec.md). Built-in functions, such as `print`
-and `circle`, are documented in the [Built-ins section](builtins.md).
+The following examples will help you understand the syntax of Evy. We also have
+a one page [Cheat Sheet](papers/evy-cheat-sheet.pdf) that you can print and
+keep handy.
+
+For a more formal definition of the syntax, see the[Language Specification]
```
Comment (camh-) [inline docs/syntax-by-example.md:7]: missing space before `[Language Specification]`.

Also, can you break the link between the text and the link? Github is not underlining the link like it is for the others.

File: README.md [hunk 2]
```diff
@@ -44,15 +44,15 @@ Here is a "hello world" program in Evy
 
 Here are some resources for learning more about Evy:
 
-- [Learn and Practise]\: A guided tour that teaches you programming using Evy.
 - [Syntax by Example](docs/syntax-by-example.md): A collection of examples that illustrate the Evy syntax.
 - [Built-in Documentation](docs/builtins.md): Details on built-in functions and events in Evy.
 - [Language Specification](docs/spec.md): A formal definition of the Evy syntax and language.
+- [Interactive Labs]\: A guided tour with challenges that teaches you programming using Evy.
```
[the remark this answers is not in the document]
Comment (juliaogris) [reply README.md:50]: it's to disambiguate from the reference providing the URL below.

[the remark this answers is not in the document]
Comment (juliaogris) [reply README.md]: does not seem to be necessary inside list. will remove.

File: docs/syntax-by-example.md (hunk 1 above)
Comment (juliaogris) [reply docs/syntax-by-example.md]: thank you. sorry for that. sloppy

**Review by camh- — APPROVED** [review]
:green_book:

**Pull request merged**